### DISCUSSION TITLE
What is this error in VSCode?

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#445] [Q&A] [answered]
Every time I compile, I keep seeing this error message in the corner of my eye
![image](https://github.com/user-attachments/assets/9db43102-4ece-4544-990a-50c6a10ddaac)
It never does anything and nothing breaks because of this error. Honestly this isn't a bug or a whatever you call it it's just a curiosity thing.

**Comment by r6915ee** [comment]
From what I'm researching, it's pretty common. I don't exactly understand why it happens in full detail, though.

**Comment by NexIsDumb** [comment, marked as answer]
yeah pretty common, maybe try to set the compile target as windows in vsc if you didnt already

**Reply by NexIsDumb** [reply]
ig it fucking actually worked😭
